Sources/Misc/Deprecations.swift

```diff
@@ -143,6 +143,12 @@ public extension Purchases {
         )
     }
 
+    /**
+     * Enable automatic collection of Apple Search Ads attribution. Defaults to `false`.
+     */
+    @available(*, deprecated, message: "Use Purchases.automaticAdServicesAttributionTokenCollection instead")
```

Comment:
Instead of `message` this would be better with `renamed` so it provides a fix-it (it's also missing the new `attribution`):
```suggestion
    @available(*, deprecated, renamed: "attribution.automaticAdServicesAttributionTokenCollection")
```
I think that should be correct but you can check it in the API testers.